README.md

```diff
@@ -1 +1,184 @@
-# Isaac-portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Isaac-portfolio] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Portfolio website]** is a website that shows my skills in web development. It has links to my projects, resume and social media.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.quackit.com/html/codes/html_code_library.cfm">Html</a></li>
+    <li><a href="https://www.w3docs.com/course/css-the-complete-guide-2020-incl-flexbox-grid-sass">css</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Navbar]**-> a navigation bar.
+- **[header]**-> general introduction to the programmer
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+ Git and Code editor.
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone git@github.com:gaks1/Isaac-portfolio.git
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Great job updating your README as suggested by the previous reviewer but there is still one more thing you're missing. This section of your README is missing some headers please refer to the [Microverse readme template](https://github.com/microverseinc/readme-template) and do well to add them :+1:

Reply:
@NahnahAJ , Hello, on this part, some of the headers seam redundant such as install, run, deployment. As neither of those are required in the project. I still don't understand their purpose on this particular project. What information would I add there?

Reply:
Yes you’re right but this project will be gradually developed and you’ll eventually add those features so it is just standard procedure that you write out all the methods and steps necessary in your README file. I hope this answers your question @gaks1

Reply:
So I should add them even even if there blank? If so will this be the standard in all future projects as well? @NahnahAJ

Reply:
For HTMl and css projects and some JavaScript projects they all follow the same installation and usage. It is only the tests and deployment you can leave blank for now but I will suggest that you include them in your readme and make sure to change them to suit future projects when necessary @gaks1